accname/name/comp_name_from_content.html

```diff
@@ -258,7 +258,39 @@ <h1 data-expectedlabel="Call us" data-testname="heading name from content with t
 <h1 data-expectedlabel="CALL US" data-testname="heading name from content with text-transform:uppercase" class="ex" style="text-transform:uppercase;">Call us</h1>
 <h1 data-expectedlabel="Call Us" data-testname="heading name from content with text-transform:capitalize" class="ex" style="text-transform:capitalize;">Call us</h1>
 <h1 data-expectedlabel="call us" data-testname="heading name from content with text-transform:lowercase" class="ex" style="text-transform:lowercase;">Call us</h1>
-  
+
+<!-- Giacomo Petri first test, be patient if there is any mistake :D -->
+<h1>Name from content containing element with implicit term role</h1>
+<p>This test aims to verify the computed acc name of elements (buttons, headings, links) containing a term role.</p>
+<p>This test covers:</p>
+<ul>
+  <li>Elements with implicit button, heading, link roles containing element with implicit term role</li>
+  <li>Elements with implicit button, heading, link roles containing element with explicit term role</li>
+  <li>Elements with explicit button, heading, link roles containing element with implicit term role</li>
+  <li>Elements with explicit button, heading, link roles containing element with explicit term role</li>
+</ul>
+<p>Tests the <a href="https://www.w3.org/TR/accname-1.2/#comp_name_from_content">Computation steps</a> (specifically point F) from the Accessible Name and Description Computation 1.2 specifications for that specific instance.</p> 
```

Comment:
This can be an HTML comment rather than a rendered paragraph.

It should also specifically mention the `term` role since that the edge case this new set of tests is covering, right?

Reply:
Also never reference the point by the outline alphanumeric, since that changes sometimes... so instead of (F)(iv) you'd reference the named anchor _**Name from Each Child**_. (#comp_name_from_content_for_each_child)

https://w3c.github.io/accname/#comp_name_from_content_for_each_child

Reply:
```suggestion
<!-- Tests `Name from Each Child` edge case failures with an embedded `term` as a child element role. -->
<!-- https://w3c.github.io/accname/#comp_name_from_content_for_each_child -->
```